packages/squiggle-lang/src/rescript/Reducer/Reducer_MathJs/Reducer_MathJs_Parse.res

```diff
@@ -2,63 +2,27 @@
   MathJs Nodes
   We make MathJs Nodes strong-typed
 */
-module AE = Reducer_Extra_Array
-module JsG = Reducer_Js_Gate
-module Rerr = Reducer_Error
+module Extra = Reducer_Extra
```

Comment:
Are you planning to refactor the Reducer_Extra into ``E.res``, or are you submitting as is? (It would be easy for me to refactor later, too)

Reply:
That works